site/docs/aliyun.md

```diff
@@ -0,0 +1,145 @@
+upper-layer data analysis engines. Aliyun DLF is a good choice to manage the apache iceberg tables, the Aliyun DLF
+catalog integration will come in the next releases.
+
+### Engines Access.
+
+All the engines (Spark, Hive, Flink, Presto) can access the iceberg table backed in aliyun services. There are following
+examples to show how to access it.
+
+The ideal way to show the example is using Aliyun DLF Catalog to manage those iceberg tables, but we still don't finish
+the iceberg + DLF integration work in apache iceberg repository. Here we are showing the examples to manage iceberg
+tables in Hive Catalog.
+
+### Spark
+
+For example, to access apache iceberg tables stored in alibaba object storage service with Apache Spark 3.2.x:
+
+```bash
+# Add Iceberg dependency
+ICEBERG_VERSION=0.13.0
+ALIYUN_ACCESS_KEY_ID=******      # Your Aliyun access key id.
+ALIYUN_ACCESS_KEY_SECRET=******  # Your Aliyun access key secret.
+ALIYUN_OSS_ENDPOINT=******       # Your Aliyun OSS endpoint.
+
+DEPENDENCIES="org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:$ICEBERG_VERSION"
+DEPENDENCIES+=",org.apache.iceberg:iceberg-aliyun-runtime:$ICEBERG_VERSION"
+
+# Start Spark SQL client shell
+spark-sql --packages $DEPENDENCIES \
+    --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions \
+    --conf spark.sql.catalog.my_catalog=org.apache.iceberg.spark.SparkCatalog \
+    --conf spark.sql.catalog.my_catalog.uri=thrift://<host>:<port> \
+    --conf spark.sql.catalog.my_catalog.warehouse=oss://my-bucket/my/key/prefix \
+    --conf spark.sql.catalog.my_catalog.io-impl=org.apache.iceberg.aliyun.oss.OSSFileIO \
+    --conf spark.sql.catalog.my_catalog.oss.endpoint=$ALIYUN_OSS_ENDPOINT \
+    --conf spark.sql.catalog.my_catalog.client.access-key-id=$ALIYUN_ACCESS_KEY_ID \
+    --conf spark.sql.catalog.my_catalog.client.access-key-secret=$ALIYUN_ACCESS_KEY_SECRET
+```
+
+Let's create iceberg tables and insert few records into it.
+
+```sql
+CREATE TABLE my_catalog.default.sample (
+    id    BIGINT,
+    data  STRING
+)
+USING iceberg
+TBLPROPERTIES (
+  'engine.hive.enabled' = 'true'
+);
+
+INSERT INTO `hive`.`default`.`sample` VALUES (1, 'AAA');
+```
+
+### Flink
+
+Take the sample that accessing apache iceberg tables stored in aliyun object storage service with Apache Flink 1.13.2: 
+
+```bash
+ICEBERG_VERSION=0.13.0
+wget $ICEBERG_MAVEN_URL/iceberg-flink-runtime/$ICEBERG_VERSION/iceberg-flink-runtime-$ICEBERG_VERSION.jar
+
+./bin/sql-client.sh embedded \
+  -j /path/to/flink-sql-connector-hive-2.3.6_2.12-1.13.2.jar \
+  -j /path/to/iceberg-aliyun-runtime-$ICEBERG_VERSION.jar \
+  -j /path/to/iceberg-flink-1.13-runtime-$ICEBERG_VERSION.jar \
+  shell
+```
+
+Let's create iceberg tables and insert few records into it.
+
+```sql
+CREATE CATALOG hive WITH (
+    'type' = 'iceberg',
+    'uri' = 'thrift://localhost:9083',
+    'warehouse' = 'oss://my-bucket/my-object',
```

Comment:
The way placeholders are displayed here is inconsistent. How about using the following?
```
<OSS_PATH>
<THRIFT-URI>
<OSS_ENDPOINT>
<ALIYUN_ACCESS_KEY_ID>
<ALIYUN_ACCESS_KEY_SECRET>
```